rules/windows/registry/registry_set/registry_set_persistence_event_viewer_events_asp.yml

```diff
@@ -12,6 +12,9 @@ date: 2023/02/17
 modified: 2023/03/05
 tags:
     - attack.persistence
+    - attack.defense_evasion
+    - attack.t1547.001
```

Comment:
This is `Boot or Logon Autostart Execution: Registry Run Keys / Startup Folder` and these keys are not related to autorun keys.
```suggestion
```